src/OpenTelemetry.Exporter.Console/OpenTelemetry.Exporter.Console.csproj

```diff
@@ -5,17 +5,13 @@
     <Description>Console exporter for OpenTelemetry .NET</Description>
     <PackageTags>$(PackageTags);Console;distributed-tracing</PackageTags>
     <MinVerTagPrefix>core-</MinVerTagPrefix>
+    <ReferenceSystemTextJsonPackages>true</ReferenceSystemTextJsonPackages>
```

Comment:
nit: This seems funky to me and may be a slippery slope.

If I didn't know anything about this repo, I would first look at the PackageReferences to tell me about project dependencies and may not catch the significance of this item. I personally prefer the ItemGroup with Condition instead of the extra Targets file.

Reply:
Only a nit pick. I don't feel strongly enough to block the PR.
I understand why it was done and I don't disagree with the intention to keep things simple.
For me, I'm thinking about times I've needed to troubleshoot csproj files and something like this likely would have tripped me up.